### DISCUSSION TITLE
Membangun Web Server Multi Aplikasi - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#22] [Comments]
# Membangun Web Server Multi Aplikasi - Tulisan Programmer Gadungan

Bismillah .... Baru kesempatan nulis lagi, karena belakang ini load pembuatan modul sekaligus aplikasi (UTDRS/ Mini PMI) ditambah Setup…

https://topidesta.my.id/server-ubuntu-aapanel/

**Comment by mdestafadilah** [comment]
Enaknya pake aapanel biar kita belum subscribe/ langganan pro version, mereka rela loh kasih support.
